NASA_CMS_2023/Mexico/DPS_algorithm/run_GMBpredictions_MEX.sh

```diff
@@ -16,4 +16,4 @@ Projects="input/Projects.gpkg"
 
 OUTPUTDIR="${PWD}/output"
 
-Rscript ${basedir}/GMB_Mexico.R ${output_csv} ${mexico} ${ccirast} ${heirast} ${data} ${INLA_model_fit_v2} ${MGN2020_INEGI_Urban_Mex_mask_DISS} ${All_Products_Comp_Over30_Binary_6933} ${Projects} 
\ No newline at end of file
+conda run --name r --no-capture-output Rscript ${basedir}/GMB_Mexico.R ${output_csv} ${mexico} ${ccirast} ${heirast} ${data} ${INLA_model_fit_v2} ${MGN2020_INEGI_Urban_Mex_mask_DISS} ${All_Products_Comp_Over30_Binary_6933} ${Projects} 
```

Comment:
minor: add a newline to the end of this file

Reply:
also added to the build_command.sh